kpops/pipeline_generator/pipeline.py

```diff
@@ -248,11 +248,9 @@ def print_yaml(self, substitution: dict | None = None) -> None:
         syntax = Syntax(
             substitute(str(self), substitution), "yaml", background_color="default"
```

Comment:
Don't we need the `word_wap=True` anymore? 🤔

Reply:
while word wrap is neat for narrow terminals, the output is then no longer valid YAML. (for deserialization)
That's why I had to change it.

Reply:
Oh, I thought we must set that.  I thought the `Console(width=1000)` was ignored by the `CliRunner()`

Reply:
It is not ignored, we just unfortunately can't set the terminal size for the `CliRunner` in the tests, which would make more sense imo. At least I didn't find a way to do so, I tried using click click `context_settings` on the `Typer` app and `$TERMINAL_SIZE` environment variable.